src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -0,0 +1,449 @@
+        {
+            string path = IOServices.RemoveTrailingSlash(file.Path);
+
+            Assert.Throws<IOException>(() =>
+            {
+                Directory.CreateDirectory(path);
+            });
+        }
+    }
+
+
+    [Fact]
+    public static void CreateDirectory_FileWithTrailingSlashAsPath_ThrowsIOException()
+    {
+        using (TemporaryFile file = new TemporaryFile())
+        {
+            string path = IOServices.AddTrailingSlashIfNeeded(file.Path);
+
+            Assert.Throws<IOException>(() =>
+            {
+                Directory.CreateDirectory(path);
+            });
+        }
+    }
+
+    [Fact]
+    public static void CreateDirectory_ExistingDirectoryWithoutTrailingSlashAsPath_DoesNotThrow()
+    {
+        using (TemporaryDirectory directory = new TemporaryDirectory())
+        {
+            string path = IOServices.RemoveTrailingSlash(directory.Path);
+
+            DirectoryInfo result = Directory.CreateDirectory(path);
+
+            Assert.Equal(path, result.FullName);
+            Assert.True(Directory.Exists(result.FullName));
+        }
+    }
+
+    [Fact]
+    public static void CreateDirectory_ExistingDirectoryWithTrailingSlashAsPath_DoesNotThrow()
+    {
+        using (TemporaryDirectory directory = new TemporaryDirectory())
+        {
+            string path = IOServices.AddTrailingSlashIfNeeded(directory.Path);
+
+            DirectoryInfo result = Directory.CreateDirectory(path);
+
+            Assert.Equal(path, result.FullName);
+            Assert.True(Directory.Exists(result.FullName));
+        }
+    }
+
+    [Fact]
+    public static void CreateDirectory_DotWithoutTrailingSlashAsPath_DoesNotThrow()
+    {
+        DirectoryInfo result = Directory.CreateDirectory(TestInfo.CurrentDirectory + @"\."); // "Current" directory
+
+        Assert.True(Directory.Exists(result.FullName));
+    }
+
+    [Fact]
+    public static void CreateDirectory_DotWithTrailingSlashAsPath_DoesNotThrow()
+    {
+        DirectoryInfo result = Directory.CreateDirectory(TestInfo.CurrentDirectory + @"\.\"); // "Current" directory
+
+        Assert.True(Directory.Exists(result.FullName));
+    }
+
+    [Fact]
+    public static void CreateDirectory_DotDotWithoutTrailingSlashAsPath_DoesNotThrow()
+    {
+        DirectoryInfo result = Directory.CreateDirectory(TestInfo.CurrentDirectory + @"\..");    // "Parent" of current directory
+
+        Assert.True(Directory.Exists(result.FullName));
+    }
+
+    [Fact]
+    public static void CreateDirectory_DotDotWithTrailingSlashAsPath_DoesNotThrow()
+    {
+        DirectoryInfo result = Directory.CreateDirectory(TestInfo.CurrentDirectory + @"\..\");    // "Parent" of current directory
+
+        Assert.True(Directory.Exists(result.FullName));
+    }
+
+#if !TEST_WINRT // Cannot set current directory to root from appcontainer with it's default ACL
+    /*
+    [Fact]
+    [ActiveIssue(0)] // SetCurrentDirectory
```

Comment:
Need something here other than just ActiveIssue(0).  Either file an issue or delete the test.